src/raps/actions/depositCompound.js

```diff
@@ -10,14 +10,19 @@ import store from '../../redux/store';
 import {
   compoundCETHABI,
   compoundCERC20ABI,
+  ethUnits,
   savingsAssetsListByUnderlying,
 } from '../../references';
 import { gasUtils, logger } from '../../utils';
 
 const NOOP = () => undefined;
-const SAVINGS_ERC20_DEPOSIT_GAS_LIMIT = 350000;
 const SAVINGS_ETH_DEPOSIT_GAS_LIMIT = 200000;
```

Comment:
should we move this one  to the ethUnits obj? I wasn't aware of its existance until you showed it to me